Public/Src/Deployment/NugetPackages.dsc

```diff
@@ -122,6 +122,54 @@ namespace NugetPackages {
         }).deployment
     });
 
+    const xldbnetcorequalifier : BuildXLSdk.DefaultQualifier = {

```

Comment:
to other reviewers: We have a little bit of cleanup to do here and can reuse it with others. I'll give it some thought and organlize later.